registration/registration-services/src/main/java/io/mosip/registration/service/operator/impl/UserOnboardServiceImpl.java

```diff
@@ -509,7 +510,7 @@ private Map<String, Object> getIdaAuthResponse(Map<String, Object> idaRequestMap
 
 			LOGGER.info(LOG_REG_USER_ONBOARD, APPLICATION_NAME, APPLICATION_ID, "Getting Symmetric Key.....");
 			// Symmetric key alias session key
-			KeyGenerator keyGenerator = KeyGeneratorUtils.getKeyGenerator("AES", 256);
+			KeyGenerator keyGenerator = KeyGeneratorUtils.getKeyGenerator("AES", 256, new SecureRandom());
```

Comment:
Can we initialize the secureRandom once and reuse it? @mahammedtaheer  what do you suggest?

Reply:
Done.